graph/service.go

```diff
@@ -11,7 +11,6 @@ import (
 func (s *TagStore) Install(eng *engine.Engine) error {
 	for name, handler := range map[string]engine.Handler{
 		"image_inspect": s.CmdLookup,
-		"image_export":  s.CmdImageExport,
 		"viz":           s.CmdViz,
 		"load":          s.CmdLoad,
```

Comment:
mmm why load and push are here if master haven't them (https://github.com/docker/docker/blob/master/graph/service.go)